src/meta/processors/sessionMan/SessionManagerProcessor.cpp

```diff
@@ -47,6 +47,9 @@ void CreateSessionProcessor::process(const cpp2::CreateSessionReq& req) {
 void UpdateSessionsProcessor::process(const cpp2::UpdateSessionsReq& req) {
     folly::SharedMutex::WriteHolder wHolder(LockUtils::sessionLock());
     std::vector<kvstore::KV> data;
+    std::unordered_map<nebula::SessionID,
+                       std::unordered_map<nebula::ExecutionPlanID, cpp2::QueryDesc>>
```

Comment:
Could use an alias here 
`using ExecPlanIdQueryDescMap = std::unordered_map<nebula::ExecutionPlanID, cpp2::QueryDesc>`

Reply:
There's not much necessary to do this since we use it not that much.